ODR issue in EsriReader

Carrying on from https://github.com/PDAL/PDAL/issues/3836:
```
$ git checkout 2.8-maintenance
$ CFLAGS="-O2 -flto -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing" CXXFLAGS="-O2 -flto -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing" LDFLAGS="-O2 -flto -Werror=odr -Werror=lto-type-mismatch -Werror=strict-aliasing" cmake -B build
$ make -C build -j$(nproc) -l$(nproc)
[...]
[ 54%] Linking CXX shared library lib/libpdalcpp.so
/tmp/PDAL/io/EsriReader.hpp:63:16: error: type ‘struct EsriReader’ violates the C++ One Definition Rule [-Werror=odr]
   63 | class PDAL_DLL EsriReader : public Reader, public Streamable
      |                ^
/tmp/PDAL/io/EsriReader.hpp:63:16: note: a different type is defined in another translation unit
   63 | class PDAL_DLL EsriReader : public Reader, public Streamable
      |                ^
/tmp/PDAL/io/EsriReader.hpp:70:39: note: the first difference of corresponding definitions is field ‘m_arbiter’
   70 |     std::unique_ptr<arbiter::Arbiter> m_arbiter;
      |                                       ^
/tmp/PDAL/io/EsriReader.hpp:70:39: note: a field of same name but different type is defined in another translation unit
   70 |     std::unique_ptr<arbiter::Arbiter> m_arbiter;
      |                                       ^
/usr/lib/gcc/x86_64-pc-linux-gnu/16/include/g++-v16/bits/unique_ptr.h:271:11: note: type name ‘std::unique_ptr<pdal::arbiter::Arbiter, std::default_delete<pdal::arbiter::Arbiter> >’ should match type name ‘std::unique_ptr<arbiter::Arbiter, std::default_delete<arbiter::Arbiter> >’
  271 |     class unique_ptr
      |           ^
lto1: some warnings being treated as errors
lto-wrapper: fatal error: /usr/bin/c++ returned 1 exit status
compilation terminated.
```

--- a/io/EsriReader.hpp
+++ b/io/EsriReader.hpp
@@ -50,13 +50,14 @@
 
 #include <nlohmann/json.hpp>
 
+namespace pdal
+{
+
 namespace arbiter
 {
     class Arbiter;
 }
 
-namespace pdal
-{
 class SrsTransform;
 class ThreadPool;
 

